docker/config/torrc-example

```diff
@@ -1,23 +1,27 @@
-##=================== /etc/torrc =====================##
-# see /usr/local/etc/tor/torrc.sample and https://www.torproject.org/docs/tor-manual.html.en
-
 # Run Tor as a regular user (do not change this)
 User atord
 DataDirectory /var/lib/tor
 
-# Server's public IP Address (usually automatic)
-#Address 10.10.10.10
+#Optional for NYX
+ControlSocket /run/tor/control
+ControlSocketsGroupWritable 1
+CookieAuthentication 1
+CookieAuthFile /run/tor/control.authcookie
+CookieAuthFileGroupReadable 1
+
+# Logging
+Log notice file /etc/tor/notices.log
```

Comment:
Please add permission changes for this file in docker-entrypoint, tor user has no access to this file by default